Source/Core/Core/HW/WiimoteReal/WiimoteReal.cpp

```diff
@@ -724,9 +725,9 @@ static bool TryToConnectWiimoteN(Wiimote* wm, unsigned int i)
 {
 	if (WIIMOTE_SRC_REAL & g_wiimote_sources[i] && !g_wiimotes[i])
 	{
+		wm->Prepare(i);
```

Comment:
As far as i can tell, IOWakeUp is implemented as signal on every platform. So that doesn't do any harm, when no one is waiting for it.